src/main/java/com/commercetools/sync/shoppinglists/ShoppingListSync.java

```diff
@@ -329,9 +356,14 @@ private CompletionStage<Void> applyCallbackAndCreate(
         .orElseGet(() -> CompletableFuture.completedFuture(null));
   }
 
-  private void handleError(@Nonnull final SyncException syncException, final int failedTimes) {
+  private void handleError(
```

Comment:
Please make this method general.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/850/commits/1f42e8a88ab662a10548d9e5d31c7554ad7fa9c8